apps/yarn.lock

```diff
@@ -1738,6 +1744,14 @@ chalk@^0.5.0, chalk@^0.5.1, chalk@~0.5.1:
     strip-ansi "^0.3.0"
     supports-color "^0.2.0"
 
+chalk@^2.0.1:
```

Comment:
I think that's the correct behavior: `yarn.lock` is freezing the dependency version unless we ask to `johnny-five` to also update dependencies.

Should I have calk@latest for `johnny-five` resolve to 2.0.1?